pkg/trace/api/endpoints.go

```diff
@@ -70,6 +70,7 @@ var endpoints = []endpoint{
 	{
 		Pattern: "/v0.5/stats",
 		Handler: func(r *HTTPReceiver) http.Handler { return http.HandlerFunc(r.handleStats) },
+		Hidden:  true,
```

Comment:
this isn't enough as the 0.77 java tracer tries to Post straight to the endpoint and does not use the info table

Reply:
It requires both changes:
1. don't register the api endpoint
2. remove the sampling enable tag from info endpoint (supposes that stats are enable)

Reply:
If we want to disable it, let's just comment it out with a note or simply remove it from the list of `endpoints`. WDYT?

Reply:
Yes, I want both...
- removal (delete or comment out)
- not in list of endpoints from info

That's what's need to present a consistent interface to the tracers.
Looks like this PR does just that, so looks good to me.